apps/emacs/emacs.talon

```diff
@@ -0,0 +1,320 @@
+app: emacs
+app: Emacs
+-
+tag(): user.tabs
+tag(): user.splits
+tag(): user.line_commands
+
+
+# ----- GENERAL ----- #
+#suplex: key(ctrl-x)
+exchange: key(ctrl-x ctrl-x)
```

Comment:
I also like to have a `quit: key(ctrl-g)` here. that's probably my most used thing I noticed missing trying this config out.

Reply:
Heh, I said the same thing today when I was reviewing. Missed this somehow :)